Modules/Timer/Timer/StartEdit/StartEditReducer.swift

```diff
@@ -5,6 +5,7 @@ import RxSwift
 import Repository
 import OtherServices
 
+// swiftlint:disable function_body_length
 // swiftlint:disable cyclomatic_complexity
```

Comment:
These two can go in the same line
```suggestion
// swiftlint:disable cyclomatic_complexity function_body_length
```